ci/docker_build_artifacts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This should be called from top repository folder.
+# It runs the `rake artifact:all` command in a docker image that has the required
+# dependencies.
+#
+# The artifacts will be placed in the build/ folder.
+
+docker build -f rakelib/Dockerfile.artifacts -t ls-artifacts rakelib/
+docker run --rm \
+	-v $(pwd):/logstash:delegated \
+	ls-artifacts \
+	sh -c "cd logstash && rake artifact:all"
```

Comment:
I think the reason this breaks on docker is that we don't have a gradle task for it. If we had `./gradlew artifactAll` invoke the rake task it would probably not segfault. Here's an example gradle task that invokes rake `https://github.com/elastic/logstash/blob/master/build.gradle#L176`

Reply:
Indeed, that does the trick, I'll update the PR.